docs/index.md

```diff
@@ -0,0 +1,6 @@
+# Documentation
```

Comment:
I think this file needs to be named `readme.md` so that it's automatically rendered?

As there's nothing here:
https://github.com/alphagov/submit-prototype-kit/tree/add-missing-sections-to-editor-docs/docs

Reply:
Quite right, sorry, I thought it was more like a web server.